src/Detail/components/BigLecueNote/BigLecueNote.style.ts

```diff
@@ -30,7 +30,7 @@ export const BigLecueNoteContentWrapper = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
-  overflow: scroll;
+  overflow: hidden auto;
```

Comment:
ASK: 특정 축만 스크롤 생기도록 하는 overflow-axis 속성이 있는데 이렇게 해도 가로 스크롤이 생길까요?!
```suggestion
  overflow-y: scroll;
```

Reply:
P5: overflow의 auto 속성을 찾아보니까 필요한 방향에만 스크롤이 생긴다고 하네요..! 그래서 한번 적용해본 결과, x축에 hidden 속성을 작성하지 않아도 스크롤이 생기지 않고 현재 저 `hidden auto`와 동일한 결과가 나타난답니다! 반영하는건 루밍이 마음!!

```suggestion
  overflow: auto;
```

Reply:
y에만 auto 속성을 주는게 가장 좋은 방법인 것 같아요! 아무래도 hidden은 기존 속성을 강제로 막는 느낌이라 빼주겠습니당
overflow 기본 속성을 scroll로 주면 내용이 넘치지 않는 경우, 즉 스크롤이 필요하지 않은 경우에도 기본으로 스크롤이 떠있어요!
그래서 `overflow: auto`로 바꿔주었던 것입니당~!!

`overflow-y: auto`를 주면 x축 스크롤에 영향을 주지 않을텐데, 전체 overflow에 auto를 주면 x축, y축 모두에 적용이 됩니다!
그래서 노트 내부 내용의 길이가 노트 높이를 넘어갈 경우, x 축에도 스크롤이 생기게 됩니당!